lfs/lfs.go

```diff
@@ -16,10 +16,6 @@ import (
 	"github.com/rubyist/tracerx"
 )
 
-const (
-	Version = "1.5.0"
```

Comment:
I think I'd rather have the version stay here than in `config`, since it makes more sense to say `lfs.Version` than `config.Version` and have to look at your import paths to realize that the `config` package comes from `github.com/git-lfs/git-lfs`.